.github/workflows/build.yml

```diff
@@ -111,6 +111,9 @@ jobs:
         run: make smelly
       - name: Check limited ABI symbols
         run: make check-limited-abi
+      - name: Check for Unsupported C Global Variables
+        if: github.event_name == 'pull_request'  # $GITHUB_EVENT_NAME
+        run: python3 Tools/c-analyzer/check-c-globals.py --format summary --traceback
```

Comment:
For consistency with the other checks, perhaps worth making this a `make` target and calling it from there?

Reply:
done